tests/e2e/config/jest.config.js

```diff
@@ -45,15 +44,17 @@ if ( process.env.E2E_GROUP ) {
 		allowedPaths.push( e2ePaths[ process.env.E2E_GROUP ] );
 	}
 } else {
+	// The 'atomic' folder is a temporary measure until we are able to run all E2E tests for Atomic sites.
```

Comment:
Did we remove the folder mentioned here in subsequent commits?

Reply:
Yep